masterlist.yaml

```diff
@@ -5587,6 +5590,26 @@ plugins:
     url: [ 'https://www.nexusmods.com/fallout4/mods/27019' ]
     after: [ 'Z_Horizon.esp' ]
 
+###### FROST ######
+  - name: 'FROST.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/18898/' ] 
+    dirty:
+      - <<: *reqManualFix
+        crc: 0xE8D519A5
+        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 1935
+        udr: 1971
+        nav: 5
+  - name: 'aFrostMod.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/48847' ]
+    dirty:
+      - <<: *reqManualFix
+        crc: 0x5284F2AD
+        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 255
+        udr: 7
+        nav: 52
```

Comment:
Since it's from a different mod page, a separate pull request like https://github.com/loot/fallout4/pull/183 is preferable.

Reply:
Done, i'll reopen the aFrostMod.esp PR once FROST is added to the masterlist.